src/legacy-routes.js

```diff
@@ -33,10 +34,11 @@ const combinedBalance = (transactions, addresses) => {
 const txToAddressInfo = (row) => ({
   ctbId: row.hash,
   ctbTimeIssued: moment(row.time).unix(),
-  ctbInputs: row.inputs_address.map(
-    (addr, i) => [addr, { getCoin: row.inputs_amount[i] }]),
-  ctbOutputs: row.outputs_address.map(
-    (addr, i) => [addr, { getCoin: row.outputs_amount[i] }]),
+  // return null if empty to distinguish delegations
```

Comment:
Don't we have certificate info as part of the table?

Reply:
I guess "distinguish" was a wrong word, I meant 'allow' since they are null and map fails.